api/src/controllers/lottery.controller.ts

```diff
@@ -163,4 +164,20 @@ export class LotteryController {
       mapTo(User, req['user']),
     );
   }
+
+  @Get(`lotteryTotals/:id`)
```

Comment:
nit pick: I'm not a fan of passing id params like this and instead think we should be passing these in as DTOs sorta how the listing controller's `getPaginatedSet` call works 

I dunno if you should change this now but that's the standard I'd really like us to move to so we don't have to remember to add the param validation on a bunch of endpoints

Reply:
Okay definitely makes sense

Reply:
Sounds good! If that feels blocking here I am happy to update that now.